saleor/graphql/order/types.py

```diff
@@ -208,11 +208,13 @@ class Meta:
             "id",
             "is_shipping_required",
             "product_name",
+            "variant_name",
             "product_sku",
             "quantity",
             "quantity_fulfilled",
             "tax_rate",
             "translated_product_name",
+            "translated_variant_name",
```

Comment:
I think we should add a description to the `translated_[...]` fields. It will be misleading for API users, they most likely would expect getting a response in their current language.